src/test/java/uk/gov/companieshouse/officerssearch/subdelta/search/AppointmentsApiClientTest.java

```diff
@@ -67,6 +69,7 @@ class AppointmentsApiClientTest {
     @BeforeEach
     void setup() {
         when(clientSupplier.get()).thenReturn(apiClient);
+        when(apiClient.getHttpClient()).thenReturn(httpClient);
```

Comment:
There are other lines of stubbing in this class which could be moved to the setup() method also. For example line 79 and 80, and the other tests those lines appear in.

Reply:
Half the tests use one version of that stubbing and half use another version, did try to do this originally when I was on ticket but had to back it out as I couldn't think how to get it to work easily.